src/dmd/tokens.d

```diff
@@ -312,6 +300,18 @@ enum TOK : int
     max_,
 }
 
+// Assert that all token enum members have consecutive values and
+// that none of them overlap
+static assert(() {
```

Comment:
This will get executed with every compiler invocation, correct?  Can we maybe put it in a `unittest` block (or is there some other solution)?

Reply:
Don't know what I was thinking.  This will only be executed when DMD is built.  It's good!

Reply:
Every time dmd is "compiled", but not when the resulting compiler is invoked.

Reply:
Clever - nice work!